modules/cnf-managing-ha-nrop-scheduler.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+:_mod-docs-content-type: CONCEPT
+[id="cnf-managing-ha-nrop_{context}"]
+= Managing high availability (HA) for the NUMA-aware scheduler
+
+The NUMA Resources Operator manages the high availability (HA) of the topology-aware secondary scheduler based on how the `spec.replicas` field is configured in the `NUMAResourcesScheduler` Custom Resource (CR). Control scheduler behavior by using one of three available options.
+
+--
+:FeatureName: Managing high availability
+include::snippets/technology-preview.adoc[]
+--
+
+[discrete]
+[id="default-ha-mode_{context}"]
+== Default high-availability mode
+
+If the `spec.replicas` field is not specified in the `NUMAResourcesScheduler` custom resource (CR), the NUMA Resources Operator automatically enables HA mode. In this mode, the Operator creates one scheduler replica for each control-plane node in the cluster to ensure redundancy.
```

Comment:
> the Operator creates one scheduler replica for each control-plane node in the cluster to ensure redundancy.

One per ctl-plane but we're limiting it to 3 by default.
 (usually there are 3 ctl-planes, but just in case there are more, we provided some limit because we don't want to overuse resources)